resources/snakefiles/assemble.smk

```diff
@@ -151,13 +152,16 @@ rule metaquast:
 
 rule multiqc_metaquast:
     input:
-        lambda wildcards: expand("output/assemble/metaquast/{sample}/report.html",
+        lambda wildcards: expand("output/assemble/{assembler}/metaquast/{sample}/report.html",
+                                 assembler=config['assemblers'],
                                  sample=samples)
     output:
-        "output/assemble/multiqc_metaquast/multiqc.html"
+        "output/assemble/{assembler}/multiqc_metaquast/multiqc.html"
```

Comment:
This needs to not have the `{assembler}` in the output, because you're expanding `config['assembers']` rather than inheriting from a wildcard. Same for log etc in this rule.

Reply:
gotcha. Those changes have been made.

Reply:
Have they been pushed? This is still showing up as `output/assemble/{assembler}/multiqc_metaquast/multiqc.html`

Reply:
wait, so do I need to checkout this branch before I add, commit and push? I thought I did this right, but it told me it was already up to date. The changes I made are showing up here:
https://github.com/CUMoellerLab/sn-mg-pipeline/pull/44/commits/a8e3a51c727cb2c69e6a8f25a7222cc89d0e3356